.ci/job_matrix_debug.yaml

```diff
@@ -39,6 +39,7 @@ runs_on_dockers:
   - {file: '.ci/Dockerfile.ubuntu16-4', name: 'ubuntu16-4', tag: 'latest', arch: 'x86_64'}
   - {name: 'blackduck', url: '$blackduck_url', category: 'tool', arch: 'x86_64', build_args: 'bobo=$var2'}
   - {url: '$registry_host/swx-storage/$arch/centos75-nvme-snap-3.0', name: 'snap', arch: 'aarch64'}
+  - {url: 'harbor.mellanox.com/toolbox/header_check:latest', name: 'header-check', tag: 'latest', category: 'header-check', arch: 'x86_64'}
```

Comment:
```category:tools``` - it will indicate to ci-demo that steps should be skipped for this container unless explicitly specified in containerSelector (by name)